sam.c

```diff
@@ -1582,6 +1582,9 @@ static bool cmd_write(Vis *vis, Win *win, Command *cmd, const char *argv[], Sele
 		return false;
 
 	Text *text = file->text;
+	Filerange range_all = text_range_new(0, text_size(text));
```

Comment:
Bellow you use `text_object_entire`, to be more consistent I would both times use the same construct.

Which one doesn't really matter, I would probably chose the explicit `text_range_new`.

Reply:
Done.